Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -885,6 +885,8 @@ WebIDL::ExceptionOr<void> XMLHttpRequest::send(Optional<DocumentOrXMLHttpRequest
                 if (!request->done()) {
                     m_timed_out = true;
                     m_fetch_controller->terminate();
+
+                    handle_errors().release_value_but_fixme_should_propagate_errors();
```

Comment:
I believe this is better to `MUST` we are in an asynchronous context here and our implementation will only throw in a synchronous context. Can also drop the extra line break.

```suggestion
                    MUST(handle_errors())
```